src/Frontend/Modules/Mailmotor/Form/SubscribeType.php

```diff
@@ -102,12 +106,14 @@ public function getInterests()
             $mailMotorInterests = $this->subscriber->getInterests();
 
             // Has interests
-            if (!empty($mailMotorInterests)) {
+            if (!empty($mailMotorInterests) && is_array($mailMotorInterests)) {
```

Comment:
But if you get errors along the way of fixing your initial bug, it is normal to also fix them